src/intl/messages.js

```diff
@@ -537,12 +544,23 @@ export const messages: { [messageId: string]: MessageType } = {
   },
   refreshInterval: 'Refresh Interval',
   refreshIntervalTooltip: 'Interval at which each screen gets refreshed.',
+  remapCtrlAltDelete: {
+    message: 'Maps Ctrl+Alt+Del shortcut to Ctrl+Alt+End',
+    description: 'Tooltip for ctrAltEnd switch',
+  },
   remoteDesktop: 'Remote Desktop',
   remoteViewerConnection: 'Remote Viewer Connection',
   remoteViewerAvailable: 'Remote Viewer is available for most operating systems. To install it, search for it in GNOME Software or run the following:',
   remove: 'Remove',
   removeVm: 'Remove VM',
   removeVmQustion: 'Are you sure you want to remove the VM? All of the data attached to the VM will be removed as well.',
+  reset: {
+    message: 'Reset',
+    description: 'Label for the Reset button',
+  },
+  resetSettings: 'Reset settings',
+  resetSettingsQuestion: 'Are you sure you want to reset your account settings?',
+  resetSettingsWarning: 'Resetting your account settings will clear all of the individual settings that were applied to it. Your account settings will be returned to the default settings. Please confirm that you want to reset your account settings before making these changes.',
```

Comment:
I would add:
"default settings (**installation defaults**)"
since those defaults are the hardcoded ones...

Reply:
> default settings (installation defaults)
>

IMHO `installation defaults` sounds more complex then simple `default settings`. I would stick to the original design here.

Reply:
> > "Automatic console connection" "automatically connect to the console..." - is missing
> 
> Yes. We have already `connectAutomatically` label which seems to be used before. Auto-connect is also a VM specific settings (not part of the Account Settings).
> 

let's change to the labels suggested by Laura.


> > account settings tabs (general, console, notifications...) are missing but we're still not sure about those so let's leave it out for now. The only Ttabs we can be sure of are maybe "general" and "console" so maybe adding only those (?)
> 
> We have already labels for: General, Console, Notifications and Advanced options
> 

+1

> > Please consider handling https://bugzilla.redhat.com/show_bug.cgi?id=1950323 on the way..
> 
> This will require gerrit patch as it's Web Admin specific.

Right, my mistake.